src/include/OpenImageIO/fmath.h

```diff
@@ -897,7 +897,7 @@ inline T safe_pow (T x, T y) {
     // less than FLOAT_MAX (but still bigger than anyone is likely to
     // need) so that subsequent additions or multiplications are less
     // likely to overflow and end up with an Inf right afterwards.
-    const T big = T(1e30);
+    const T big = std::numeric_limits<T>::max();
```

Comment:
Any reason to change this constant? The comment above would have to be updated ...